python/ray/dag/compiled_dag_node.py

```diff
@@ -848,26 +857,39 @@ def _get_or_compile(
             visited.add(cur_idx)
 
             task = self.idx_to_task[cur_idx]
-            # Create an output buffer for the actor method.
-            assert task.output_channel is None
-
             type_hint = task.dag_node.type_hint
             if type_hint.requires_nccl():
                 type_hint.set_nccl_group_id(self._nccl_group_id)
 
             if isinstance(task.dag_node, ClassMethodNode):
+                # Create output buffers for the actor method.
```

Comment:
Did you mean output channels?

Reply:
The original comments use "buffer" instead of "channel" in this file.